bedrock/firefox/tests/test_views.py

```diff
@@ -223,7 +223,7 @@ def test_send_android_sms_with_country(self):
         })
         ok_(resp_data['success'])
         self.mock_send_sms.assert_called_with('post', 'subscribe_sms', data={
-            'mobile_number': '15558675309',
+            'mobile_number': '5558675309',
```

Comment:
Just making sure it's okay to send a number to basket without a country code.

Reply:
Yup. There's new code in basket that interprets the number along with the country code and will format appropriately.